docs/troubleshoot/faq.md

```diff
@@ -0,0 +1,64 @@
+# Frequently asked questions (FAQ)
+
+This section outlines the most frequently asked questions (FAQs) about Percona Everest.
+
+
+## What is DatabaseEngine?
+
+The `DatabaseEngine` is a resource that defines the compatible PG/PXC/MongoDB versions for the current Percona Everest version. 
+
+You can use this resource to query and update the list of supported engine versions.
+
+Refer to our [API documentation](https://percona-everest.readme.io/reference/getkubernetesclusterresources-1) for usage information.
+
+## Do we have logs of the API calls made?
+
+Yes, the Percona Everest backend has logs. It runs as a `percona-everest` deployment within the `everest-system` namespace.
```

Comment:
It was `everest-server` deployment in the previous release, did the naming convention change by any chance?